.circleci/config.yml

```diff
@@ -104,6 +104,12 @@ jobs:
           root: packages
           paths:
             - "*"
+  markdown:
```

Comment:
You'll also want to add this to the jobs below! Sorry - forgot to mention this earlier!